capstone/src/main/webapp/script.js

```diff
@@ -259,6 +259,14 @@ function drawBarChart(toxicityData) {
 
 /** Shows the avaiable attributes given a language selected on text analyzer page */
 function showAvailableAttributes() {
+  $('.checkbox-menu').on('change', "input[type='checkbox']", function() {
+    $(this).closest('li').toggleClass('active', this.checked);
```

Comment:
Is it possible you can make use of this checked CSS selector, so you don't need this callback?  https://www.w3schools.com/cssref/sel_checked.asp

Reply:
The problem I had was assigning a style to an `<li>` element when its child `<input>` element got checked or unchecked. I'll check out the documentation to see if this is possible without the callback.

Reply:
It still looks like jQuery is the way to go here from what I've found

Reply:
SGTM, can you just add comments to this callback (and others in the related dropdown PRs) just explaining this?  Thanks

Reply:
Will do - they've been added.